### DISCUSSION TITLE
Any recommendations for implementing vertical waveform in newest version?

### DISCUSSION THREAD

**Discussion by baileyhamersly** [katspaugh/wavesurfer.js#3715] [Q&A]
I am in need of a waveform that is displayed vertically and on researching I see that there used to be an out-of-the-box option to display waveform vertically but it has since been deprecated. Any insight or tips on trying to recreate this in the newest version?

**Comment by katspaugh** [comment]
It’s indeed not implemented in the new version, but you can use v6 which is still available.

**Reply by khemarato** [reply]
Unfortunately v6 isn't compatible with the React library :(
